libcxx/docs/ReleaseNotes/19.rst

```diff
@@ -77,6 +77,8 @@ Improvements and New Features
 Deprecations and Removals
 -------------------------
 
+- The status of the C++03 implementation will be frozen. From now on only bug fixes will be back-ported to C++03.
```

Comment:
Can you elaborate what are *not* backported? IMO that's more interesting to the public.

Reply:
I also think that it's confusing to document this here. It should only be under LLVM 21, otherwise it provides the impression that we're freezing something in LLVM 19.